tests/__snapshots__/optionalCatchAll.test.js.snap

```diff
@@ -8,5 +8,6 @@ exports[`Routing creates Netlify redirects 1`] = `
 /page  /.netlify/functions/next_page  200
 /  /.netlify/functions/next_all  200
 /_next/*  /_next/:splat  200
-/*  /.netlify/functions/next_all  200"
+/*  /.netlify/functions/next_all  200
+/*  /:splat  200"
```

Comment:
This is the issue we are discussing on Slack rn :neutral_face:

Reply:
For future reference: I thought that having these two redirects would cause a problem and incorrectly assumed that the redirect engine used for cookie-based redirects does not support shadowing. That assumption is incorrect. I do not understand the effect that `/*  /:splat  200` is having, but everything is working correctly only when that redirect is present (including when having a root-level optional-catch-all route). I just added a Cypress test for optional catch all at root level to confirm.

Marking as resolved :heavy_check_mark: